diff --git a/node_modules/@capacitor/browser/android/build.gradle b/node_modules/@capacitor/browser/android/build.gradle
index 1fad7dc..092b3bf 100644
--- a/node_modules/@capacitor/browser/android/build.gradle
+++ b/node_modules/@capacitor/browser/android/build.gradle
@@ -43,7 +43,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
     lintOptions {
