diff --git a/node_modules/@capacitor/browser/android/build.gradle b/node_modules/@capacitor/browser/android/build.gradle
index 1fad7dc..dae50b9 100644
--- a/node_modules/@capacitor/browser/android/build.gradle
+++ b/node_modules/@capacitor/browser/android/build.gradle
@@ -1,3 +1,4 @@
+
 ext {
     capacitorVersion = System.getenv('CAPACITOR_VERSION')
     junitVersion = project.hasProperty('junitVersion') ? rootProject.ext.junitVersion : '4.13.2'
@@ -43,7 +44,7 @@ android {
     buildTypes {
         release {
             minifyEnabled false
-            proguardFiles getDefaultProguardFile('proguard-android.txt'), 'proguard-rules.pro'
+            proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
         }
     }
     lintOptions {
